packages/apps/shopify-app-remix/src/server/authenticate/admin/scope/request.ts

```diff
@@ -0,0 +1,22 @@
+import {Session} from '@shopify/shopify-api';
+
+import type {BasicParams} from '../../../types';
+import {redirectToInstallPage} from '../helpers/redirect-to-install-page';
+
+export function requestScopesFactory(params: BasicParams, session: Session) {
+  return async function requestScopes(scopes: string[]) {
+    const {logger, api} = params;
+
+    logger.debug('Request scopes: ', {
```

Comment:
Total nit, but is this clearer?
```suggestion
    logger.debug('Request optional scopes: ', {
```